services/api/src/models/Users.test.ts

```diff
@@ -73,9 +74,10 @@ describe('users', () => {
       .catch(done.fail);
     });
 
-    it('returns valid results for players with quest history', (done: DoneFn) => {
+    fit('returns valid results for players with quest history', (done: DoneFn) => {
```

Comment:
typo with the `f`?

Reply:
Good catch - `fit` actually restricts the test to only the unit tests with the `f` prefix... was using it to debug that particular line. Removed.